core/dmap.asm

```diff
@@ -0,0 +1,190 @@
+PUSH1 0x80
+CALLVALUE
+PUSH2 0x6B
+JUMPI
+PUSH1 0x1F
+PUSH2 0x15C
+CODESIZE
+DUP2
+SWAP1
+SUB
+SWAP2
+DUP3
+ADD
+PUSH1 0x1F
+NOT
+AND
+DUP4
+ADD
+SWAP2
+PUSH1 0x1
+PUSH1 0x1
+PUSH1 0x40
+SHL
+SUB
+DUP4
+GT
+DUP5
+DUP5
+LT
+OR
+PUSH2 0x70
+JUMPI
+DUP1
+DUP5
+SWAP3
+PUSH1 0x20
+SWAP5
+PUSH1 0x40
+MSTORE
+DUP4
+CODECOPY
+DUP2
+ADD
+SUB
+SLT
+PUSH2 0x6B
+JUMPI
+MLOAD
+PUSH1 0x1
+PUSH1 0x1
+PUSH1 0xA0
+SHL
+SUB
+DUP2
+AND
+DUP2
+SUB
+PUSH2 0x6B
+JUMPI
+PUSH1 0x1
+PUSH1 0xFF
+SHL
+PUSH1 0x0
+SSTORE
+PUSH1 0x60
+SHL
+PUSH1 0x1
+SSTORE
+PUSH1 0x40
+MLOAD
+PUSH1 0xD5
+SWAP1
+DUP2
+PUSH2 0x87
+DUP3
+CODECOPY
+RETURN
+JUMPDEST
+PUSH1 0x0
+DUP1
+REVERT
+JUMPDEST
+PUSH4 0x4E487B71
+PUSH1 0xE0
+SHL
+PUSH1 0x0
+MSTORE
+PUSH1 0x41
+PUSH1 0x4
+MSTORE
+PUSH1 0x24
+PUSH1 0x0
+REVERT
+INVALID
+DEPLOY
+// deployed code start
+CALLDATASIZE
+PUSH1 0x24
+EQ
+PUSH1 0x8A
+JUMPI
+CALLDATASIZE
+PUSH1 0x64
+EQ
+PUSH1 0x12
+JUMPI
+PUSH1 0x0
+DUP1
+REVERT
+JUMPDEST
+PUSH1 0x4
+CALLDATALOAD
+PUSH1 0x24
+CALLDATALOAD
+PUSH1 0x44
+CALLDATALOAD
+SWAP2
+PUSH1 0x0
+SWAP3
+CALLER
+DUP5
+MSTORE
+DUP2
+PUSH1 0x20
+MSTORE
+DUP1
+DUP4
+PUSH1 0x40
+DUP7
+KECCAK256
+SWAP4
+CALLER
+DUP8
+DUP1
+LOG4
+PUSH1 0x1
+DUP3
+ADD
+SSTORE
+DUP1
+SLOAD
+PUSH32 0x8000000000000000000000000000000000000000000000000000000000000000
+AND
+ISZERO
+PUSH1 0x86
+JUMPI
+PUSH1 0x4
+DUP4
+PUSH32 0xA4F0D7D000000000000000000000000000000000000000000000000000000000
+DUP2
+MSTORE
+REVERT
+JUMPDEST
+SSTORE
+DUP1
+RETURN
+JUMPDEST
+PUSH1 0x1
+PUSH1 0x4
+CALLDATALOAD
+DUP1
+SLOAD
+PUSH1 0x0
+MSTORE
+ADD
+SLOAD
+PUSH1 0x20
+MSTORE
+PUSH1 0x40
+PUSH1 0x0
+RETURN
+INVALID
+LOG2
+PUSH5 0x6970667358
+0x22
+SLT
+KECCAK256
+SELFBALANCE
+GAS
```

Comment:
w t f is this shit
what are they putting in the compiler

Reply:
Maybe swarm hash

Reply:
And/or ipfs CID